ci/bamboo/build-osx.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -o errexit
+set -o xtrace
+
+# Fixup $PATH for --user installation
+export PATH=${HOME}/Library/Python/2.7/bin:${PATH}
+export PYTHONPATH=${HOME}/Library/Python/2.7/lib/python/site-packages:${PATH}
```

Comment:
D'oh!  Yep.  Though, it seems to be working as is as I assume that $PYTHONPATH is already empty.